attack.go

```diff
@@ -178,6 +180,7 @@ func attack(opts *attackOpts) (err error) {
 		vegeta.KeepAlive(opts.keepalive),
 		vegeta.Connections(opts.connections),
 		vegeta.HTTP2(opts.http2),
+		vegeta.HTTP3(opts.http3),
```

Comment:
I believe this option needs to come before `TLSConfig` as to not override the side effects that opt has.